### ISSUE TITLE
Various animation / particals are green.

### ISSUE THREAD

**Issue by die-fi** [ValveSoftware/Dota-2#926]
Since a few days a lot of animations have a green glow to them instead of intended colors. 

Here are a few examples:
Enigmas Black Hole: ![20160628041419_1](https://cloud.githubusercontent.com/assets/12123373/16402167/c16ca9d2-3cea-11e6-8838-8ea593db2d89.jpg)
Color of Teleportation Scroll ![20160628041759_1](https://cloud.githubusercontent.com/assets/12123373/16402137/725f1960-3cea-11e6-90df-74bfe3d42bbf.jpg)
Naix footprints: ![20160628041805_1](https://cloud.githubusercontent.com/assets/12123373/16402135/725b6702-3cea-11e6-9c3f-d722ebd674bd.jpg)
Euls Cyclon animation: ![20160628042212_1](https://cloud.githubusercontent.com/assets/12123373/16402139/72615db0-3cea-11e6-812c-1aa4490a7e42.jpg)
Bane moving particles: ![20160628042237_1](https://cloud.githubusercontent.com/assets/12123373/16402136/725ed31a-3cea-11e6-9d19-a152d8fac80a.jpg)

I think you get the gist of it.

My system is the following:

```
dpkg -l |grep steam
ii  steam                                                       1.0.0.49-1                           i386         Valve's Steam digital software delivery system

Using Beta Client
Build:          Jun 14 2016, at 23:24:07
Steam API:      v017
Steam package version:  1465948400

My system:
ATI Moblitiy Radeon HD 4650
2xMonitors setup(1920x1080 & 1600x900)
Debian Jessie
GNOME 3.14.1

dpkg -l | grep radeon
ii  libdrm-radeon1:amd64                                        2.4.58-2                             amd64        Userspace interface to radeon-specific kernel DRM services -- runtime
ii  libdrm-radeon1:i386                                         2.4.58-2                             i386         Userspace interface to radeon-specific kernel DRM services -- runtime
ii  radeontool                                                  1.6.3-1                              amd64        utility to control ATI Radeon backlight functions on laptops
ii  radeontop                                                   0.8-1                                amd64        Utility to show Radeon GPU utilization
ii  xserver-xorg-video-radeon                                   1:7.5.0-1                            amd64        X.Org X server -- AMD/ATI Radeon display driver

dpkg -l | grep mesa
ii  glx-alternative-mesa                                        0.7.3~bpo8+1                         amd64        allows the selection of MESA as GLX provider
ii  libegl1-mesa:amd64                                          11.1.3-1~bpo8+1                      amd64        free implementation of the EGL API -- runtime
ii  libegl1-mesa-dev:amd64                                      11.1.3-1~bpo8+1                      amd64        free implementation of the EGL API -- development files
ii  libegl1-mesa-drivers:amd64                                  11.1.3-1~bpo8+1                      amd64        transitional dummy package
ii  libgl1-mesa-dev:amd64                                       11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL API -- GLX development files
ii  libgl1-mesa-dri:amd64                                       11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL API -- DRI modules
ii  libgl1-mesa-dri:i386                                        11.1.3-1~bpo8+1                      i386         free implementation of the OpenGL API -- DRI modules
ii  libgl1-mesa-glx:amd64                                       11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL API -- GLX runtime
ii  libgl1-mesa-glx:i386                                        11.1.3-1~bpo8+1                      i386         free implementation of the OpenGL API -- GLX runtime
ii  libglapi-mesa:amd64                                         11.1.3-1~bpo8+1                      amd64        free implementation of the GL API -- shared library
ii  libglapi-mesa:i386                                          11.1.3-1~bpo8+1                      i386         free implementation of the GL API -- shared library
ii  libgles1-mesa:amd64                                         11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL|ES 1.x API -- runtime
ii  libgles1-mesa-dev:amd64                                     11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL|ES 1.x API -- development files
ii  libgles2-mesa:amd64                                         11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL|ES 2.x API -- runtime
ii  libgles2-mesa-dev:amd64                                     11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL|ES 2.x API -- development files
ii  libglu1-mesa:amd64                                          9.0.0-2                              amd64        Mesa OpenGL utility library (GLU)
ii  libglu1-mesa:i386                                           9.0.0-2                              i386         Mesa OpenGL utility library (GLU)
ii  libglu1-mesa-dev                                            9.0.0-2                              amd64        Mesa OpenGL utility library -- development files
ii  libopenvg1-mesa:amd64                                       10.3.2-1+deb8u1                      amd64        free implementation of the OpenVG API -- runtime
ii  libwayland-egl1-mesa:amd64                                  11.1.3-1~bpo8+1                      amd64        implementation of the Wayland EGL platform -- runtime
ii  mesa-common-dev:amd64                                       11.1.3-1~bpo8+1                      amd64        Developer documentation for Mesa
ii  mesa-utils                                                  8.2.0-1                              amd64        Miscellaneous Mesa GL utilities


dpkg -l | grep drm
ii  libdrm-intel1:amd64                                         2.4.58-2                             amd64        Userspace interface to intel-specific kernel DRM services -- runtime
ii  libdrm-intel1:i386                                          2.4.58-2                             i386         Userspace interface to intel-specific kernel DRM services -- runtime
ii  libdrm-nouveau2:amd64                                       2.4.58-2                             amd64        Userspace interface to nouveau-specific kernel DRM services -- runtime
ii  libdrm-nouveau2:i386                                        2.4.58-2                             i386         Userspace interface to nouveau-specific kernel DRM services -- runtime
ii  libdrm-radeon1:amd64                                        2.4.58-2                             amd64        Userspace interface to radeon-specific kernel DRM services -- runtime
ii  libdrm-radeon1:i386                                         2.4.58-2                             i386         Userspace interface to radeon-specific kernel DRM services -- runtime
ii  libdrm2:amd64                                               2.4.58-2                             amd64        Userspace interface to kernel DRM services -- runtime
ii  libdrm2:i386                                                2.4.58-2                             i386         Userspace interface to kernel DRM services -- runtime
ii  libva-drm1:amd64                                            1.4.1-1                              amd64        Video Acceleration (VA) API for Linux -- DRM runtime

dpkg -l |grep glx
ii  glx-alternative-mesa                                        0.7.3~bpo8+1                         amd64        allows the selection of MESA as GLX provider
ii  glx-diversions                                              0.7.3~bpo8+1                         amd64        prepare for using accelerated GLX implementations from GPU vendors
ii  libgl1-mesa-glx:amd64                                       11.1.3-1~bpo8+1                      amd64        free implementation of the OpenGL API -- GLX runtime
ii  libgl1-mesa-glx:i386                                        11.1.3-1~bpo8+1                      i386         free implementation of the OpenGL API -- GLX runtime
ii  libva-glx1:amd64                                            1.4.1-1                              amd64        Video Acceleration (VA) API for Linux -- GLX runtime
ii  libxcb-glx0:amd64                                           1.10-3+b1                            amd64        X C Binding, glx extension
ii  libxcb-glx0:i386                                            1.10-3+b1                            i386         X C Binding, glx extension
ii  libxcb-glx0-dev:amd64                                       1.10-3+b1                            amd64        X C Binding, glx extension, development files
ii  update-glx                                                  0.7.3~bpo8+1                         amd64        utility for switching the GLX implementation
```

**Comment by gdrewb-valve** [comment]
If you validate your Dota install from the Steam client does that help?

**Comment by die-fi** [comment]
Game cache verification didn't help.
Funny thing, while most skills are green, Undeads Decay and Pudges Rot gives of blueish/purple particles.

**Comment by danginsburg** [comment]
It looks like you are running AMD mesa 11.1.3 on AMD HD4650.  Are you able to try a different version of mesa? When we originally released 10.5.9 was known good.  Perhaps you could try 11.2.x?

**Comment by gdrewb-valve** [comment]
Is this issue still active? Were you able to try a different version of Mesa?
